helm/nifikop/templates/service_account.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if .Values.rbacEnable }}
```

Comment:
does it really have to do whether rbac is enabled or not ? 
I think you should add a parameter `serviceAccount.create` set to `true` by default in the `values.yaml` file.
And add this condition here `{{- if .Values.serviceAccount.create }}`